### ISSUE TITLE
Accessibility (WCAG Compliance)

### ISSUE THREAD

**Issue by therebelrobot** [therebelrobot/brigadehub#19]
It is imperative that this project is done with Accessibility in mind, at least for the supported themes. For that end, I propose we conform this project to [WCAG](https://www.w3.org/TR/2008/REC-WCAG20-20081211/) standards, which I'll abridge below. This could be a good opportunity to find/build a linter to check the codebase for immediate issues like `alt` tags on images, etc.
#### Principle 1: Perceivable

_Information and user interface components must be presentable to users in ways they can perceive._
- [ ] **Guideline 1.1 Text Alternatives:** Provide text alternatives for any non-text content so that it can be changed into other forms people need, such as large print, braille, speech, symbols or simpler language.
- [ ] **Guideline 1.2 Time-based Media:** Provide alternatives for time-based media.
- [ ] **Guideline 1.3 Adaptable:** Create content that can be presented in different ways (for example simpler layout) without losing information or structure.
- [ ] **Guideline 1.4 Distinguishable:** Make it easier for users to see and hear content including separating foreground from background.
#### Principle 2: Operable

_User interface components and navigation must be operable._
- [ ] **Guideline 2.1 Keyboard Accessible:** Make all functionality available from a keyboard.
- [ ] **Guideline 2.2 Enough Time:** Provide users enough time to read and use content.
- [ ] **Guideline 2.3 Seizures:** Do not design content in a way that is known to cause seizures.
- [ ] **Guideline 2.4 Navigable:** Provide ways to help users navigate, find content, and determine where they are.
#### Principle 3: Understandable

_Information and the operation of user interface must be understandable._
- [ ] **Guideline 3.1 Readable:** Make text content readable and understandable.
- [ ] **Guideline 3.2 Predictable:** Make Web pages appear and operate in predictable ways.
- [ ] **Guideline 3.3 Input Assistance:** Help users avoid and correct mistakes.
#### Principle 4: Robust

_Content must be robust enough that it can be interpreted reliably by a wide variety of user agents, including assistive technologies._
- [ ] **Guideline 4.1 Compatible:** Maximize compatibility with current and future user agents, including assistive technologies.

**Comment by andrewmbacon** [comment]
This is something I can help with, I have a lot of experience in this area. I would recommend a minimum of WCAG 2 (Level A), and it wouldn't take much more for us to also hit the revised ADA Section 508 standards. 

The real trick is designing tools that encourage users of a CMS to have their content be accessible.

**Comment by therebelrobot** [comment]
As discussed this past wednesday, this may be too big of a task to focus on for alpha release. Retagging.

**Comment by andrewmbacon** [comment]
Testing with Lynx!   The browser you run from the shell.

https://failicide.wordpress.com/2015/02/27/install-lynx-browser-on-os-x/

**Comment by therebelrobot** [comment]
related: https://code.facebook.com/posts/1152362248125846/hike-our-quick-simple-accessibility-primer/

**Comment by therebelrobot** [comment]
Issue moved to [brigadehub/theme-public-c4sf #14](https://github.com/brigadehub/theme-public-c4sf/issues/14) via [**ZenHub**](https://www.zenhub.com/)
